### PR TITLE
Issue 28 : Handle JMAP injection with Guice

### PR DESCRIPTION
**Pull request by rouazana**

### CODE REVIEW

**Comment by aduprat** [comment]
:+1:

File: server/protocols/jmap/src/main/java/org/apache/james/jmap/json/MultipleClassesDeserializer.java [hunk 1]
```diff
@@ -34,6 +34,8 @@
 
 public class MultipleClassesDeserializer extends StdDeserializer<Object> {
 
+    private static final long serialVersionUID = 1L;
```
Comment (mbaechler) [inline server/protocols/jmap/src/main/java/org/apache/james/jmap/json/MultipleClassesDeserializer.java]: you forgot to remove this one

File: server/protocols/jmap/src/main/java/org/apache/james/jmap/model/AccessTokenRequest.java [hunk 2]
```diff
@@ -18,11 +18,10 @@
  ****************************************************************/
 package org.apache.james.jmap.model;
 
-import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
```
Comment (mbaechler) [inline server/protocols/jmap/src/main/java/org/apache/james/jmap/model/AccessTokenRequest.java]: we should avoid import organize only commits

File: server/protocols/jmap/src/test/java/org/apache/james/jmap/model/ContinuationTokenTest.java [hunk 3]
```diff
@@ -21,14 +21,13 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 
+import java.time.ZonedDateTime;
```
Comment (mbaechler) [inline server/protocols/jmap/src/test/java/org/apache/james/jmap/model/ContinuationTokenTest.java]: organize only ...

File: server/protocols/jmap/src/main/java/org/apache/james/jmap/json/MultipleClassesDeserializer.java (hunk 1 above)
Comment (rouazana) [inline server/protocols/jmap/src/main/java/org/apache/james/jmap/json/MultipleClassesDeserializer.java]: ok

File: server/protocols/jmap/src/main/java/org/apache/james/jmap/model/AccessTokenRequest.java (hunk 2 above)
Comment (rouazana) [inline server/protocols/jmap/src/main/java/org/apache/james/jmap/model/AccessTokenRequest.java]: In fact it removes unused DateTimeException import

**Comment by mbaechler** [comment]
:+1:

**Comment by aduprat** [comment]
:+1:

**Pull request closed without merging**